spec/computable/registry/challenge.spec.ts

```diff
@@ -142,29 +142,29 @@ describe('Registry: Challenge', () => {
     expect(tx1).toBeTruthy()
 
     const id = eventReturnValues('_Challenge',
-      await registry.challenge(listBytes, '', { from: challenger }), 'id')
+      await registry.challenge(web3, listBytes, '', { from: challenger }), 'id')
     expect(id).toBeTruthy()
 
     // should be able to fetch the actual challenge object
     const chall:Challenge = await registry.challenges(id)
     expect(chall).toBeTruthy()
 
     // 1 serving as a truthy vote for the challenged, i.e a falsy vote and it would not be listed
-    await voting.commitVote(web3, id, voter, 1, 10, 420)
-    await increaseTime(provider, ParameterDefaults.COMMIT_STAGE_LENGTH + 1)
-    await voting.revealVote(id, 1, 420, { from: voter })
-    await increaseTime(provider, ParameterDefaults.REVEAL_STAGE_LENGTH + 1)
-    await registry.updateStatus(listBytes)
+    // await voting.commitVote(web3, id, voter, 1, 10, 420)
+    // await increaseTime(provider, ParameterDefaults.COMMIT_STAGE_LENGTH + 1)
+    // await voting.revealVote(id, 1, 420, { from: voter })
+    // await increaseTime(provider, ParameterDefaults.REVEAL_STAGE_LENGTH + 1)
+    // await registry.updateStatus(web3, listBytes)
 
-    expect(await registry.isWhitelisted(listBytes)).toBe(true)
+    // expect(await registry.isWhitelisted(listBytes)).toBe(true)
 
-    // check that the token dispensation is correct
-    const listing:RegistryListing = await registry.listings(listBytes),
-      actualUnstaked = maybeParseInt(listing.unstakedDeposit),
-      expectedUnstaked = ParameterDefaults.MIN_DEPOSIT +
-        (ParameterDefaults.MIN_DEPOSIT * ParameterDefaults.DISPENSATION_PCT / 100)
+    // // check that the token dispensation is correct
+    // const listing:RegistryListing = await registry.listings(listBytes),
+      // actualUnstaked = maybeParseInt(listing.unstakedDeposit),
+      // expectedUnstaked = ParameterDefaults.MIN_DEPOSIT +
+        // (ParameterDefaults.MIN_DEPOSIT * ParameterDefaults.DISPENSATION_PCT / 100)
```

Comment:
Is this block intentionally commented out?

Reply:
no.

lol.